src/main/resources/com/google/api/codegen/java/build_gapic.gradle.snip

```diff
@@ -21,7 +21,9 @@
     compile 'com.google.api:gax:{@metadata.gaxVersionBound.lower}'
     compile 'com.google.api:gax-grpc:{@metadata.gaxGrpcVersionBound.lower}'
     testCompile 'junit:junit:4.12'
+    testCompile 'commons-lang:commons-lang:2.6'
```

Comment:
Shouldn't this be removed?

Reply:
Done. Yes, I guess it's just really hard to actually remove commons-lang.